cms/djangoapps/import_from_modulestore/constants.py

```diff
@@ -0,0 +1,5 @@
+"""
+Constants for import_from_modulestore app
+"""
+
+IMPORT_FROM_MODULESTORE_PURPOSE = "import_from_modulestore"
```

Comment:
```suggestion
IMPORT_FROM_MODULESTORE_STAGING_PURPOSE = "import_from_modulestore"
```
Out of context, it's not clear what this constant is for. We can clarify that it's a staged content type

Reply:
Renamed